Example/Utils/Settings.swift

```diff
@@ -0,0 +1,47 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018-2019 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import Foundation
+
+class Settings: NSObject {
+
+    @objc class func setActiveTheme(theme: ThemeHelpers.ThemeName) {
+        let themeDefinition = ThemeHelpers.themeDefinition(forTheme: theme)
+
+        let themeName = themeDefinition.themeName
+
+        UserDefaults.standard.set(themeName, forKey: "theme")
+    }
+
+    @objc class func activeTheme() -> ThemeHelpers.ThemeName {
+        guard let currentTheme = UserDefaults.standard.value(forKey: "theme") as? String else {
+            return .none
+        }
+
+        switch currentTheme {
```

Comment:
This isn't required instead we can use an enum with a a raw value of `String` in combination with 

```swift
ThemeName(rawValue: currentTheme) ?? .none
```

Reply:
Yeah but Objective C 😢

Reply:
Once BPKAppDelegate has been rewritten we can do this fo sho!